src/components/avatar/avatar.tsx

```diff
@@ -92,7 +94,8 @@ export const EuiAvatar: FunctionComponent<EuiAvatarProps> = ({
     'euiAvatar',
     sizeToClassNameMap[size],
     typeToClassNameMap[type],
-    className
+    className,
+    isDisabled ? 'euiAvatar--disabled' : ''
```

Comment:
Just a suggestion for using our conventions.

```suggestion
    {
      'euiAvatar-isDisabled': isDisabled,
    },
    className
```